apps/site/src/views/Resources/sections/BackendResources/BackendResources.module.scss

```diff
@@ -0,0 +1,32 @@
+.row {
+	margin-bottom: 50px;
+}
+
+.column {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	margin-bottom: 20px;
+	gap: 120px;
+}
+
+.card {
+	background-image: url("~@/assets/index_cards/backend-description-card.png");
+	width: 1127px;
+	height: 230px;
+	position: relative;
+	display: flex;
+	justify-content: center;
+}
```

Comment:
Issue: The text overlaps on mobile view. As with the API Resources index card, we might want to avoid hardcoding width and height and setting the text to position absolute.

<img width="161" alt="image" src="https://github.com/HackAtUCI/zothacks-site-2023/assets/28026520/2eea1fe5-06a1-4767-b4da-3763d88da950">